### PR TITLE
Bin transformation

### PR DESCRIPTION
**Pull request by GraceGSy**
Changes in this PR:

More options for binning transformation
- As provided in [geostats](https://github.com/simogeo/geostats)
- See storybook for examples

Docs
- Updated binning docs
- Updated plottitle docs

Needs feedback:
- Does `UniqueValues` classification make sense for binning transformation? (Seems more categorical to me)
- Binning docs could be better organised...

### CODE REVIEW

**Comment by atepoorthuis** [comment]
> * Does `UniqueValues` classification make sense for binning transformation? (Seems more categorical to me)

Isn't this identical to a `GroupBy`?

**Comment by GraceGSy** [comment]
> > * Does `UniqueValues` classification make sense for binning transformation? (Seems more categorical to me)
> 
> Isn't this identical to a `GroupBy`?

That's what it seemed to me. Shall I just remove this then? Unless we want to make it possible to group discrete numeric values?

File: docs/transform/binning.md [hunk 1]
```diff
@@ -2,9 +2,61 @@
...
+### EqualInterval
+
+The `EqualInterval` method groups data into a given number of equal sized bins.
+
+Key       | Type      |  Description 
+----------|-----------|----------------------------
+numClasses| Number    | Number of bins
+
+### StandardDeviation, ArithmeticProgression, Geometric Progression, Quantile, Jenks
+
```
Comment (atepoorthuis) [inline docs/transform/binning.md]:
```suggestion
These classification methods are made available through [geostats](https://github.com/simogeo/geostats) (see geostats docs for details on each method). 
```

File: docs/transform/binning.md [hunk 2]
```diff
@@ -2,9 +2,61 @@
...
+The `UniqueValues` classification method does not take any additional keys.
+
+### Manual
+
+The `Manual` method groups data into user-defined ranges.
+
+Key          | Type      |  Description 
+-------------|-----------|----------------------------
+manualClasses| Array     | An array of [lowerBound, upperBound] ranges
```
Comment (atepoorthuis) [inline docs/transform/binning.md]: Does this effectively ask the user to duplicate the upperbound as lowerbound of the next bin? So: `[[0, 1], [1, 2], [2, 3]]`? Would there be anything against simplifying by only requiring the the class boundary instead? So `[0, 1, 2, 3]`. We could use `geostats.setClassManually` to do this.

Comment (GraceGSy) [reply docs/transform/binning.md]: Doesn't `geostats.setClassManually` return the given array?

But yes, I think it is a good idea for the user to specify `[0, 1, 2, 3, 4]` and we can change it into [lower, upper] for them.

Comment (atepoorthuis) [reply docs/transform/binning.md]: Cool! I guess that's why we do this review thing :)

**Pull request merged**